### PR TITLE
creates checker config

### PR DESCRIPTION
**Pull request by jenlij**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -72,7 +72,7 @@ require (
 	github.com/nats-io/go-nats v1.6.0 // indirect
 	github.com/nats-io/go-nats-streaming v0.4.0 // indirect
 	github.com/nats-io/nats v1.6.0 // indirect
-	github.com/nats-io/nats-streaming-server v0.11.0 // indirect
```
Comment (jenlij) [inline go.mod]: not sure why these changed

**Comment by jenlij** [comment]
Still needs to be tested

File: pkg/datarepair/checker/checker.go [hunk 2]
```diff
@@ -36,14 +39,30 @@ func (c Config) Run(ctx context.Context, server *provider.Provider) (err error)
...
 	defer cancel()
+	var errs []error
 
-	go func() {
+	go func() error {
 		for {
 			select {
 			case <-ticker.C:
 				zap.S().Info("Starting segment checker service")
+				go func() {
```
Comment (aleitner) [inline pkg/datarepair/checker/checker.go]: we want to separate all the code from the config Run. We want the checker to have it's own run that config uses. This gives more flexibility on how we can start the checker later. see https://github.com/storj/storj/blob/master/pkg/datarepair/repairer/repairer.go#L50

File: pkg/datarepair/checker/checker.go [hunk 3]
```diff
@@ -146,3 +165,15 @@ func lookupResponsesToNodes(responses *pb.LookupResponses) []*pb.Node {
 	}
 	return nodes
 }
+
+func combinedError(errs []error) error {
```
Comment (aleitner) [inline pkg/datarepair/checker/checker.go]: we are using combined error in a couple places in code now. Each place has it's own combined error. Move this function definition to the utils folder

Comment (egonelbre) [reply pkg/datarepair/checker/checker.go]: There's already `utils.CombineErrors`

**Pull request closed without merging**